cypress/e2e/cloud/tasklist.test.ts

```diff
@@ -0,0 +1,83 @@
+import {Organization} from '../../../src/types'
+
+describe('Checkout Page', () => {
```

Comment:
checkout page? We should be able to put this all in the `tasks.test.ts` file

Reply:
Copy Pasta